src/query/config/src/outer_v0.rs

```diff
@@ -518,6 +518,9 @@ impl From<InnerCacheConfig> for CacheConfig {
         };
 
         match inner.params {
+            StorageParams::Disabled => {
+                cfg.cache_type = "disabled".to_string();
```

Comment:
It is better to add a new cache type 'None' or `BlackHole`. It means we can write through this cache backend, the cache layer does nothing except we configure the cache.